artikel/2015-07-31-Hitzewelle-Sommerspaß-oder-Klimawandel.md

```diff
@@ -0,0 +1,34 @@
+#Hitzewelle: Sommerspaß oder Klimawandel?
+
+http://www.dw.com/de/hitzewelle-sommerspa%C3%9F-oder-klimawandel/a-18616694
+
+der Artikel "Hitzewelle: Sommerspaß oder Klimawandel?" spricht ein
+interessantes und wichtiges Thema an, weil wir diesen Sommer so viel 
+Hitzwelle in Deutschland gehabt haben.
+
+Der Artikel geht um Klimawandel und die Möglichkeit, ob es existiert. In 
```

Comment:
You want to refer to the content of the article, so it would be

    In dem Artikel geht es um

instead of

    Der Artikel geht um

There is a german pattern very similar to your construct. It is

    Der Artikel handelt von

inflected in _Dativ_.

Second part of the sentence wants to refer to the existence of climate change. This is hard, but I think I would do sth. like:

    Der Artikel handelt vom Klimawandel und dessen mögliche Existenz.